src/main/java/com/example/hw18/controller/ControllerWeb.java

```diff
@@ -0,0 +1,166 @@
+        return "allUsers";
+    }
+
+
+    @PutMapping("/newUser")
+    public @ResponseBody User createUser(@RequestBody User user) {
+
+        for (Map.Entry<User, ArrayList<Car>> item : garage.entrySet()) {
+            if (item.getKey().equals(user)) {
+                throw new ResponseStatusException(
+                        HttpStatus.CONFLICT, "user with this name is arleady exist");
+            }
+        }
+        garage.put(user, new ArrayList<Car>());
+        return user;
+    }
+
+    @PutMapping("/newCar")
+    public @ResponseBody Car createCar(@RequestBody Car car) {
+
+        for (Car car1 : cars) {
+            if (car.getCarId() == car1.getCarId()) {
+                throw new ResponseStatusException(
+                        HttpStatus.CONFLICT, "car with this id is arleady exist");
+            }
+        }
+        cars.add(car);
+        return car;
+    }
+
+    @PutMapping("/newRandomCar")
+    public @ResponseBody Car randomCar() {
+        cars.add(carService.addNewCar(cars.size()));
+        return carService.addNewCar(cars.size());
+    }
+
+    @PutMapping("/newRandomUser")
+    public @ResponseBody User randomUser() {
+        garage.put(userService.addUser(garage.size()), new ArrayList<Car>());
+        return userService.addUser(garage.size());
+
+    }
+
+    @PostMapping("/makeOvners/{userId}/{carId}")
+    public @ResponseBody User makeOvners(@PathVariable(value = "userId") int userId,
+                                         @PathVariable(value = "carId") int carId) {
+
+        garage.get(userService.findUserById(garage, userId)).add(cars.get(carId));
+
+        for (Car car : garage.get(userService.findUserById(garage, userId))) {
+            if (car.getCarId() == carId) {
+                car.setUserIdOvner(userId);
+            }
+        }
+        return userService.findUserById(garage, userId);
+    }
+
+    @DeleteMapping("/deleteCar/{carId}")
+    public @ResponseBody List<Car> deleteCar(@PathVariable(value = "carId") int carId) {
+        cars.removeIf(car -> car.getCarId() == carId);
+        return cars;
+    }
+
+    @DeleteMapping("/deleteUser/{userId}")
+    public @ResponseBody Map<User, ArrayList<Car>> deleteUser(@PathVariable(value = "userId") int userId) {
+        for (User user : garage.keySet())
+            if (user.getUserId() == userId)
+                garage.remove(user);
+        return garage;
+
+    }
+
+    @PostMapping("/updateUser/{userId}")
+    public @ResponseBody User updateUser(@RequestBody User user,
+                                         @PathVariable(value = "userId") int userId) {
+        for (Map.Entry<User, ArrayList<Car>> item : garage.entrySet()) {
+            if (item.getKey().equals(userService.findUserById(garage, userId))) {
+                item.getKey().setName(user.getName());
+            }
+            return item.getKey();
+        }
+        return null;
+    }
+
+    @PostMapping("/updateCar/{carId}")
```

Comment:
post mean it is update method so name don't need to include word update => /car/{carId}